src/main/resources/com/puppycrawl/tools/checkstyle/checks/sizes/messages_tr.properties

```diff
@@ -1,19 +1,14 @@
 #Generated by ResourceBundle Editor (http://eclipse-rbe.sourceforge.net)
-
-executableStatementCount = Yürütülebilir ifade sayısı {0,number,integer} (maksimum izin verilen değer {1,number,integer}).
-
-maxLen.anonInner = İsimsiz iç sınıf uzunluğu  {0,number,integer} satır (maksimum izin verilen değer {1,number,integer}).
-maxLen.file      = Dosya uzunluğu {0,number,integer} satır (maksimum izin verilen değer {1,number,integer}).
-maxLen.method    = Metot uzunluğu {0,number,integer} satır (maksimum izin verilen değer {1,number,integer}).
-
-maxLineLen = Satır {0,number,integer} değerinden daha uzun ({1,number,integer} değer bulundu).
-
-maxOuterTypes = Tanımlanan dış tür sayısı  {0,number,integer} (maksimum izin verilen değer {1,number,integer}).
-
-maxParam = {0,number,integer} değerinden daha fazla parametre mevcut {1,number,integer} değer bulundu).
-
-too.many.methods          = Toplam metot sayısı {0,number,integer} (maksimum izin verilen değer {1,number,integer}).
-too.many.packageMethods   = Toplam ''package'' erişimli metot sayısı {0,number,integer} (maksimum izin verilen değer {1,number,integer}).
-too.many.privateMethods   = Toplam ''private'' metot sayısı {0,number,integer} (maksimum izin verilen değer {1,number,integer}).
-too.many.protectedMethods = Toplam ''protected'' metot sayısı {0,number,integer} (maksimum izin verilen değer {1,number,integer}).
-too.many.publicMethods    = Toplam ''public'' metot sayısı {0,number,integer} (maksimum izin verilen değer {1,number,integer}).
+executableStatementCount=Yürütülebilir ifade sayısı {0,number,integer} (maksimum izin verilen değer {1,number,integer}).
+maxLen.anonInner=İsimsiz iç sınıf uzunluğu  {0,number,integer} satır (maksimum izin verilen değer {1,number,integer}).
```

Comment:
re- formatting should be done in separate commit.

Reply:
Fixed, it was automatic formatting by Intellij that I missed.